Assets/HoloToolkit/Input/Scripts/GazeManager.cs

```diff
@@ -129,5 +129,17 @@ private void UpdateStabilizationPlane()
                 HolographicSettings.SetFocusPointForFrame(newFocusPointPosition, -gazeDirection);
             }
         }
+
+        /// <summary>
+        /// Sends message to the object currently focused on.
+        /// </summary>
+        /// <param name="message">Message to send</param>
+        public void SendMessageToFocusedObject(string message)
```

Comment:
But to implement the functionality on the focused object you'd have to know that SendMessage was used correct?